src/NuGet.Core/NuGet.PackageManagement/Projects/ProjectItemProperties.cs

```diff
@@ -9,11 +9,12 @@ namespace NuGet.ProjectManagement
     /// </summary>
     public static class ProjectItemProperties
     {
-        public const string IncludeAssets = "IncludeAssets";
-        public const string ExcludeAssets = "ExcludeAssets";
-        public const string PrivateAssets = "PrivateAssets";
+        public const string IncludeAssets = nameof(IncludeAssets);
+        public const string ExcludeAssets = nameof(ExcludeAssets);
+        public const string PrivateAssets = nameof(PrivateAssets);
         public const string IsImplicitlyDefined = nameof(IsImplicitlyDefined);
         public const string NoWarn = nameof(NoWarn);
-        public const string GeneratePathProperty = "GeneratePathProperty";
+        public const string GeneratePathProperty = nameof(GeneratePathProperty);
+        public const string Aliases = nameof(Aliases);
```

Comment:
same as previous comment. I think the values should be string literals, because we don't want them changing just in case someone renames a field name in the future.

Reply:
If we are concerned with breaking chance, we won't change these. I'm just trying to follow the same pattern.

We also already do this a lot, like all over our constants code: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.PackageManagement/Projects/ProjectBuildProperties.cs 

Search of `= nameof(` in the codebase and you will see heavy use of this.